src/com/goide/GoAnnotator.java

```diff
@@ -40,6 +40,9 @@ public void annotate(@NotNull PsiElement o, @NotNull AnnotationHolder holder) {
         o.putUserData(GoReference.IMPORT_USERS, ContainerUtil.<PsiElement>newArrayListWithCapacity(0));
       }
     }
+    else if (o instanceof GoLiteral) {
+      setHighlighting(o, holder, GoSyntaxHighlightingColors.NUMBER);
+    }
```

Comment:
From the grammar (go.bnf):
```
Literal ::=
    int
  | float
  | floati
  | decimali
  | hex
  | oct
  | imaginary
  | rune
  | StringLiteral
  | char
```
So, you need to check that it's a hex or oct, but not a char or a rune.